tests/cases/setup_pemserver/inventory.yml.j2

```diff
@@ -0,0 +1,13 @@
+---
+all:
+  children:
+    primary:
+      hosts:
+        primary1:
+          ansible_host: {{ inventory_vars['primary1_ip'] }}
+          private_ip: {{ inventory_vars['primary1_ip'] }}
```

Comment:
I don't think we need a `primary` node if we are going to test PEM server only. Could you confirm?

Reply:
Does not need primary node